code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -381,3 +381,10 @@
 	R.module.add_module(S, FALSE, TRUE)
 
 	return 1
+
+
+/obj/item/borg/upgrade/ai
+	name = "ai control module"
+	desc = "Used to kick in a cyborg's VTEC systems, increasing their speed."

```

Comment:
"Used to enable ai mainframe access on borgs, allowing the ai to remotely control them"